tuxemon/resources/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -1213,3 +1213,20 @@ msgstr "Well, it looks like your Tuxemon are full of energy! Here they are!"
 
 msgid "input_name"
 msgstr "Name?"
+
+msgid "default_names"
+msgstr "Alex\n"
+"Ariel\n"
+"Francis\n"
+"Jade\n"
+"Jordan\n"
+"Lou\n"
+"Modeste\n"
+"Morgan\n"
+"Manu\n"
+"Marie\n"
+"Sasha\n"
+"Yanick"
+
+msgid "dont_care"
```

Comment:
Pedantic, but this is a different than the others.

Should be:

msgid "dont_care"
msgstr "S'en fiche."

msgid "default_names"
msgstr "Alex\n"
"Ariel\n"
"Francis\n"
"Jade\n"
"Jordan\n"
"Lou\n"
"Modeste\n"
"Morgan\n"
"Manu\n"
"Marie\n"
"Sasha\n"
"Yanick"